data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
Great job @osamaashraf6 🤝

- [ ] [Optional] Kindly ensure you maintain the values for all the `weight_kg` as floats instead of integers.